src/js/components/sharedComponents/Loading.jsx

```diff
@@ -56,15 +56,15 @@ export const LoadingWrapper = ({
                                 <FontAwesomeIcon icon="spinner" spin size="lg" />
                                 <h4>{`${msg}${dots}`}</h4>
                             </div>
-                            {includeFooter && (<Footer />)}
+                            {includeFooter && (<Footer pageName="Loading" />)}
                         </PageHeader>
                     </>
                 )}
                 <div className="page__loading">
                     <FontAwesomeIcon icon="spinner" spin size="lg" />
                     <h4>{`${msg}${dots}`}</h4>
                 </div>
-                {includeFooter && (<Footer />)}
+                {includeFooter && (<Footer pageName="Loading" />)}
```

Comment:
Does this cause analytics to fire an event each time the loading component is displayed?

Reply:
This only provides the page name in case the list sign-up is clicked on that page; shouldn't trigger analytics by just displaying.

Reply:
But I also didn't check where we use a loading indicator that includes the site footer either.

Reply:
Got it thanks, I see now trackLink fires the events👍